test/parallel/test-debug-brk-no-arg.js

```diff
@@ -1,13 +0,0 @@
-'use strict';
```

Comment:
Removed because `inspect` behaviour is different from `debug`. The former stops *before* reaching the repl. This test doesn't make sense anymore.